src/main/java/com/varukha/onlinebookstore/mapper/OrderItemsMapper.java

```diff
@@ -0,0 +1,17 @@
+package com.varukha.onlinebookstore.mapper;
+
+import com.varukha.onlinebookstore.config.MapperConfig;
+import com.varukha.onlinebookstore.dto.orderitem.OrderItemDto;
+import com.varukha.onlinebookstore.model.CartItem;
+import com.varukha.onlinebookstore.model.OrderItem;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper(config = MapperConfig.class)
+public interface OrderItemsMapper {
+    @Mapping(target = "bookId", source = "orderItem.book.id")
```

Comment:
You can simply write "book.id" instead "orderItem.book.id", result will be the same

Reply:
agree